app/components/Auth/styles.styl

```diff
@@ -1,4 +1,4 @@
-@import './../../mixins.styl';
+@import './../../styles/mixins.styl';
```

Comment:
`./` in paths is not needed if you use `../` already.
So, instead of
```javascript
@import './../../styles/mixins.styl';
```
write
```javascript
@import '../../styles/mixins.styl';
```